unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/WearableItem.cs

```diff
@@ -180,15 +213,18 @@ public int GetIssuedCountFromRarity(string rarity)
         return int.MaxValue;
     }
 
-    public string ComposeThumbnailUrl() { return baseUrl + thumbnail; }
+    public string ComposeThumbnailUrl()
+    {
+        return baseUrl + thumbnail;
+    }
 
     public static HashSet<string> ComposeHiddenCategories(string bodyShapeId, WearableItem[] wearables)
     {
         HashSet<string> result = new HashSet<string>();
         //Last wearable added has priority over the rest
         for (int index = 0; index < wearables.Length; index++)
         {
-            WearableItem wearableItem = wearables[index];
+            var wearableItem = wearables[index];
```

Comment:
Using the explicit type gives more clarity to the code, I think we should keep this as it was before.

Reply:
i modified it through IDE suggestions, but i rollbacked the change